src/test/java/org/alliancegenome/curation_api/ExpressionBulkUploadFmsITCase.java

```diff
@@ -0,0 +1,93 @@
+import org.alliancegenome.curation_api.resources.TestContainerResource;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(TestContainerResource.Initializer.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("606 - Expression bulk upload - FMS")
+@Order(606)
+public class ExpressionBulkUploadFmsITCase extends BaseITCase {
+
+	private final String expressionBulkPostEndpoint = "/api/gene-expression-annotation/bulk/ZFIN/annotationFile";
+	private final String expressionTestFilePath = "src/test/resources/bulk/fms/07_expression/";
+	private final String expressionFindEndpoint = "/api/gene-expression-annotation/find?limit=100&page=0";
+	private final String taxon = "NCBITaxon:7955";
+	private final String gene = "GEXPTEST:GENE001";
+	private final String mmoTerm = "GEXPTEST:assay001";
+	private final String referenceId = "ZFIN:PUB001";
+	private final String publicationId = "PMID:009";
+
+	@BeforeEach
+	public void init() {
+		RestAssured.config = RestAssuredConfig.config()
+			.httpClient(HttpClientConfig.httpClientConfig()
+				.setParam("http.socket.timeout", 100000)
+				.setParam("http.connection.timeout", 100000));
+	}
+
+	@Test
+	@Order(1)
+	public void expressionBulkUploadAllFields() throws Exception {
+		loadRequiredEntities();
+		checkSuccessfulBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "AF_01_all_fields.json");
+	}
+
+	@Test
+	@Order(2)
+	public void expressionBulkUploadMissingRequiredFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_01_no_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_02_no_dateAssigned.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_03_no_evidence.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_04_no_assay.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_05_no_whenExpressed.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_06_no_whereExpressed.json");
+	}
+
+	@Test
+	@Order(3)
+	public void expressionBulkUploadEmptyRequiredFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_01_empty_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_02_empty_dateAssigned.json");
+	}
+
+	@Test
+	@Order(4)
+	public void expressionBulkUploadInvalidFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_01_invalid_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_02_invalid_dateAssigned.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_04_invalid_assay.json");
+
```

Comment:
Would like more tests here to cover all fields that your validation class checks for validity

Reply:
added for reference and publication id